pkgs/development/libraries/flutter-engine/deps.nix

```diff
@@ -0,0 +1,735 @@
+{ lib, fetchgit, fetchcipd, fetchFromGitHub, hostPlatform }@args:
+with lib;
+let
+  fetchFromGitHub = { owner, repo, fetchSubmodules ? false, rev ? "HEAD", sha256 ? fakeHash }: args.fetchFromGitHub {
+    inherit owner repo fetchSubmodules rev sha256;
+  };
+
+  fetchFromGoogle = { owner, repo, fetchSubmodules ? false, rev ? "HEAD", sha256 ? fakeHash }: fetchgit {
+    url = "https://${owner}.googlesource.com/${repo}.git";
+    inherit rev sha256 fetchSubmodules;
+  };
+
+  toolchainArch = if hostPlatform.isx86_64 then "amd64" else if hostPlatform.isAarch64 then "aarch64" else throws "Unsupported platform";
+  ## Note: this must match Flutter Engine's DEPS file
+in {
+  "src/buildtools/linux-x64/clang" = fetchcipd {
```

Comment:
flutter requires clang_15, which is not-yet in nixpkgs https://github.com/NixOS/nixpkgs/pull/209536

clang_14 throws

> error: unknown warning option '-Wno-unqualified-std-cast-call'

Reply:
clang_15 throws

> clang-15: error: unable to execute command: posix_spawn failed: No such file or directory

https://github.com/milahu/nixpkgs/tree/flutter-engine/init

Reply:
flutter's clang &rarr; posix_spawn failed

nixpkgs' clang &rarr; `/nix/store/178vvank67pg2ckr5ic5gmdkm3ri72f3-binutils-2.39/bin/ld: cannot find -lstdc++: No such file or directory`

seems like compiler flags are wrong
see the `toolchain.ninja` file (where is it generated? by `gn`?)